script.js

```
@@ -0,0 +1,28 @@
+function daysInMonth(month, year) {
+  try {
+    if (!daysAmountInMonth(month, year)) {
+      throw new TypeError("exeption");
+    } 
+      return daysAmountInMonth(month, year));
+    } catch (error) {
+      return error.message;
```

`... the function should throw an exception.`